src/app/register-page/register-page.component.html

```diff
@@ -0,0 +1,22 @@
+<form>
+<div class="container">
+    <h1>Registreer een account</h1>
+    <h1 style="color:red">{{message}}</h1>
+    
+    <label>E-Mail</label>
+    <input type="text" name="mail" placeholder="Uw e-mailadres" required>
+
+    <label>Username</label>
+    <input type="text" name="username" placeholder="Uw username" required>
```

Comment:
THIS IS A DUTCH APP, WE USE GEBRUIKERSNAAM ON THE FRONT-END

![image](https://user-images.githubusercontent.com/43812715/84051178-0961d980-a9af-11ea-9441-98907a53eb5b.png)

Reply:
Still says "Uw username" instead of "Uw gebruikersnaam"

Reply:
Updated it